src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -49,6 +59,18 @@ public HttpCassandraManagementProxy(MetricRegistry metricRegistry,
     this.metricRegistry = metricRegistry;
     this.rootPath = rootPath;
     this.endpoint = endpoint;
+    // configure the Management API client
+    ApiClient api = new ApiClient()
+        // with the approrpiate base URL (i.e. "http://localhost:8080")
+        .setBasePath(endpoint.getHostString())
+        // probably want the default 10 second connection timeout
+        //.setConnectTimeout(0)
+        // At the very least, the stopNode operation can take longer than the default 10 second read timeout
+        .setReadTimeout(0)
+        // We might have write operations that take longer than 10 seconds too
+        .setWriteTimeout(0);
+    // setup the helper client with the configured api
+    this.apiClient = new DefaultApi(api);
```

Comment:
The biggest question here is getting the right Base URL for the client to connect. I don't know that `endpoint` here is going to have the protocol prefix (i.e. "http://").